Makefile

```diff
@@ -290,6 +290,10 @@ android-clean: ##@prepare Clean Gradle state
 	git clean -dxf -f ./android/app/build; \
 	[[ -d android/.gradle ]] && cd android && ./gradlew clean
 
+xcode-clean: SHELL := /bin/sh
+xcode-clean: ##@prepare Clean xcode derivation folder
+	rm -fr $(HOME)/Library/Developer/Xcode/DerivedData/StatusIm-*
```

Comment:
How about also removing `$(HOME)/Library/Developer/Xcode/Archives/*/StatusIm*`?

Reply:
You could do something like:
```Makefile
xcode-clean: SHELL := /bin/sh
xcode-clean: XCODE_HOME := $(HOME)/Library/Developer/Xcode
￼xcode-clean: ##@prepare Clean xcode derivation folder
	rm -fr $(XCODE_HOME)/DerivedData/StatusIm-* $(XCODE_HOME)/Archives/*/StatusIm*
```

Reply:
I can add `Archives`. The target was added rather to help with local simulator build because currently there is no option to clear derived data folder without opening Xcode.

Reply:
Right, but it doesn't hurt, actually helps free up space, and does what the name of the target says.

Reply:
done